schemas/com.snowplowanalytics.snowplow/pii_enrichment/jsonschema/1-0-0

```diff
@@ -0,0 +1,654 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a PII enrichment original values",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "pii_enrichment",
```

Comment:
Is there any better names for the schema? I wouldn't understand its purpose without `description` above.

Reply:
What do you suggest?

Reply:
`pii_original` maybe? Or `pii_values`

Reply:
pii_original_values sounds better to me but I think @alexanderdean was thinking that in the future this may be used for more things (e.g. unique customer identifier).

Reply:
@knservis's idea here was `pii_identification` - what do you think @chuwy ? We are looking for something that captures the action that led to the event, the moment in time - not just the description of the bundle of data we are sending (especially as that bundle could change). Other options `pii_transformation`, `pii_...`?

Reply:
I think `pii_transformation` is definitely step forward. `pii_capture`? `pii_snapshot` is also about description of data, but sounds good.

Reply:
I think pii_capture is the best of those (pii_cleartext_mapping is another idea), let's not drag this too long. @alexanderdean ?

Reply:
Let's go with `pii_transformation` for now.